### PR TITLE
Images for review

### PR DESCRIPTION
**Pull request by dorileo**
Changes proposed in this pull request:
- Add the following image descriptors: kvm, legacy-kvm, live-docker,
live, provision, service-os

### CODE REVIEW

**Comment by dorileo** [comment]
@mdhorn updated adding vmware image, I'm dropping for now the provision image - I could not find the current users for this image.

**Comment by dorileo** [comment]
@mdhorn just updated the image descriptors, please note I'm adding the live-desktop image, and also see that I'm still installing these files in the post-install script:

```
cp themes/clr.png $THEMES_DIR
cp etc/clr-installer.desktop $DESKTOP_DIR
cp etc/custom.conf $GDM_DIR
cp scripts/clr-installer-desktop.sh $CHROOTPATH/usr/bin/
```

I want the ```clr.png``` to be installed by the package installing the gnome wallpaper, the ```.desktop``` and the ```clr-installer-desktop.sh``` files must be installed by the ```clr-installer``` package+bundle.

**Comment by dorileo** [comment]
@mdhorn please see 52c157edfe9fb07bb6294f198fc870dec2876ff6

File: etc/clr-installer.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=Install Clear Linux OS
+Comment=Clear Linux OS Installer
+Icon=/usr/share/clr-installer/themes/clr.png
+Exec=/usr/bin/clr-installer-desktop.sh
+Terminal=false
+Categories=Tags;Describing;Application
```
Comment (mdhorn) [inline etc/clr-installer.desktop]: Can you add a newline to this?

File: scripts/live-desktop.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+targetMedia:
+- name: ${bdevice}
+  size: "8G"
+  type: disk
+  children:
+  - name: ${bdevice}1
+    fstype: vfat
+    mountpoint: /boot
+    size: "64M"
```
Comment (mdhorn) [inline scripts/live-desktop.yaml]: Up to 150M?

**Pull request merged**